quic/s2n-quic-tls-default/Cargo.toml

```diff
@@ -17,11 +17,7 @@ s2n-quic-tls = { version = "=0.32.0", path = "../s2n-quic-tls" }
 s2n-quic-rustls = { version = "=0.32.0", path = "../s2n-quic-rustls" }
 
 [target.'cfg(unix)'.dev-dependencies]
-# newer versions require rust 1.66, see https://github.com/aws/s2n-quic/issues/1991
-# this version pin is only needed to prevent verification failures when using
-# cargo package / cargo publish, as those commands do not respect the version pin
-# in downstream dev-dependencies (in s2n-quic-tls, in this case)
-jobserver = "=0.1.26"
+jobserver = "0.1"
```

Comment:
This should be removed as well

Reply:
Ok you meant most of the these were used to pin not just `serde_spanned`. I'll do another sweep.

Reply:
yeah anything pinned is likely a transitive dependency